py/server/deephaven/table.py

```diff
@@ -61,11 +61,15 @@ def __init__(self, j_table: jpy.JType):
 
     def __repr__(self):
         default_repr = super().__repr__()
+        # default_repr is in a format like so:
+        # deephaven.table.Table(io.deephaven.engine.table.Table(objectRef=0x7f07e4890518))
+        # We take the last two brackets off, add a few more details about the table, then add the necessary brackets back
         column_dict = {col.name: col.data_type for col in self.columns[:10]}
         repr_str = (
             f"{default_repr[:-2]}, num_rows = {self.size}, columns = {column_dict}"
         )
-        repr_str = repr_str[:115] + "...}>" if len(repr_str) > 120 else repr_str
+        # We need to add the two brackets back, and also truncate it to be 120 char total (118 + two brackets)
+        repr_str = repr_str[:114] + "...}))" if len(repr_str) > 118 else repr_str + "))"
```

Comment:
Is there an associated unit test that needs changing?  If not, it seems like there should be one. @jmao-denver

Reply:
we do have a test case for repr() (test_repr) that can be used to test this specific change.